app/src/main/java/com/stevesoltys/seedvault/transport/backup/BackupCoordinator.kt

```diff
@@ -430,14 +432,23 @@ internal class BackupCoordinator(
 
     private fun getBackupBackoff(): Long {
         val noBackoff = 0L
-        val defaultBackoff = DAYS.toMillis(30)
+        val longBackoff = DAYS.toMillis(30)
 
         // back off if there's no storage set
-        val storage = settingsManager.getStorage() ?: return defaultBackoff
-        // don't back off if storage is not ejectable or available right now
-        return if (!storage.isUsb || storage.getDocumentFile(context).isDirectory) noBackoff
-        // otherwise back off
-        else defaultBackoff
+        val storage = settingsManager.getStorage() ?: return longBackoff
+
+        // back off if storage is removable and not available right now
+        return if (storage.isUsb && !storage.getDocumentFile(context).isDirectory) longBackoff
+        // back off if storage is on network, but we have no access
+        else if (storage.requiresNetwork && !hasInternet()) HOURS.toMillis(1)
```

Comment:
Any reason for picking an hour?

Reply:
No, just seems like a reasonable time when internet could be back.